### PR TITLE
Added getLeafs function

### PR DESCRIPTION
**Pull request by khaledkhamis**

### CODE REVIEW

**Comment by AdrianSkierniewski** [comment]
Hi,

Thank you for creating this PR.
Could you add some tests to prove that this functionality works as expected?

**Comment by khaledkhamis** [comment]
Sure!

**Comment by khaledkhamis** [comment]
done!

File: src/Gzero/EloquentTree/Model/Tree.php [hunk 1]
```diff
@@ -348,7 +348,7 @@ public static function getRoots()
      *
      * @return \Illuminate\Database\Eloquent\Builder
      */
-    public static function getLeafs()
+    public static function getLeaves()
```
Comment (AdrianSkierniewski) [inline src/Gzero/EloquentTree/Model/Tree.php]: It looks like both forms are valid :)

**Comment by AdrianSkierniewski** [comment]
Maybe you could add one additional test with only root node without children. In this case getLeafs() should return root node it self, right? Is it expected behaviour in this case?

**Comment by AdrianSkierniewski** [comment]
overall it looks good for me :+1:

**Comment by khaledkhamis** [comment]
Just added the test case for a tree with one node (root)

**Comment by AdrianSkierniewski** [comment]
@khaledkhamis merged & released

**Pull request merged**